bundles/org.openhab.binding.sinope/README.md

```diff
@@ -13,20 +13,20 @@ Right now, only the thermostat devices (3000W and 4000W) (TH1120RF) are supporte
 ## Discovery
 
 The Sinopé Gateway (bridge) discovery is not supported for now.
-It will be added in future release.
-The Sinopé devices discovery is implemented.
+It will be added in a future release.
```

Comment:
Empty promise, the binding hasn't been maintained for six years - see https://github.com/openhab/openhab-addons/pulls?q=is%3Apr+in%3Atitle+sinope+is%3Aclosed:
```suggestion
```